this_the_way.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+
+# the function read the path and create list of the files start with deep, then print them
+def thisTheWay():
+    folder_path = input("insert the path to the folder")
+    directory_list = os.listdir(folder_path)
+    only_deep = [f for f in directory_list if f.startswith("deep")]
+    print(only_deep)
```

Comment:
Notice that the function is supposed to return the list, and not to print it.
After fixing it, remember to fix your documentation too, and add return type to the function.

Reply:
what is the meaning of "add a return type to the function"?
do u mean to add the return type to the documentation?